cmd/root.go

```diff
@@ -40,8 +40,9 @@ import (
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 
-	// Adding explicitely the GCP auth plugin
+	// Explicitly adds the GCP and OIDC auth plugins
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
```

Comment:
probably it is better to do:
```
_ "k8s.io/client-go/plugin/pkg/client/auth
```
so that include the plugin for Azure as well

Reply:
oh, you're right - didn't see that the top-level package includes them all